src/report/report.dto.ts

```diff
@@ -1,13 +1,14 @@
 import { FilterableField, IDField } from '@nestjs-query/query-graphql';
-import { ObjectType, GraphQLISODateTime, ID } from '@nestjs/graphql';
+import pkg from '@nestjs/graphql';
+const { ObjectType } = pkg;
```

Comment:
I think you could rewrite this as
```
import pkg, {ObjectType} from '@nestjs/graphql';
```

Not that this doesn't work, but it's probably cleaner to avoid destructuring the import after the fact if we can avoid it

Reply:
you'd think but no:

`> rm -rf dist/ && tsc && cp package.json dist/

(node:26304) ExperimentalWarning: Importing JSON modules is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
file:///Users/mattraffel/src/kiva/reporting/dist/report/report.dto.js:11
import pkg, { ObjectType } from '@nestjs/graphql';
              ^^^^^^^^^^
SyntaxError: Named export 'ObjectType' not found. The requested module '@nestjs/graphql' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from '@nestjs/graphql';
const { ObjectType } = pkg;`

Reply:
Yeah, you can't destructure a CommonJS module on import. Only default import is allowed, for whatever reason. The other option, if you want to go that way, is to just do the default import and then use dot annotation to get the specific module member whenever you want it. Like what you did below with `pkg.ID`.

The one bit of feedback I'd give is that `pkg` is a very misleading name. If I were reading this code, I would think it referred to the local package. A better name would be `nestJsGraphql` or something like that.